lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2727,6 +2727,9 @@ def print_pdf(self, filename, *,
             else:            # we opened the file above; now finish it off
                 file.close()
 
+    def draw(self, *args, **kwargs):
+        _no_output_draw(self.figure)
+        return super().draw(*args, **kwargs)
```

Comment:
Why did you pass `*args, **kwargs` here, but not in the other cases?

Reply:
Contrary to our phone conversation, the bug was passing args/kwargs.  The draw method on canvases should take no arguments!